fail2ban/tests/servertestcase.py

```diff
@@ -158,6 +158,9 @@ class Transmitter(TransmitterBase):
 	def setUp(self):
 		self.server = TestServer()
 		super(Transmitter, self).setUp()
+		# so far isStarted only tested but not used otherwise
+		# and here we don't really .start server
+		self.assertFalse(self.server.isStarted())
```

Comment:
Only one test case will fail in error case...

Additionally, have you seen how fast the test cases are now? :)
Never do something unnecessarily, I've just added one line above and this code checks only once the server not started in setup... (see ad393fcfbbdb0c491e86f891c6ee194fa0134f79)